build/HookManConfig.cmake

```diff
@@ -29,7 +29,7 @@ if(NOT WIN32)
   message(STATUS "Found GCC base directory: ${GCC_BASE_DIR}")
 
   set(CMAKE_CXX_FLAGS       "-Wall")
-  set(CMAKE_CXX_LINK_FLAGS  "-lstdc++")
+  set(CMAKE_CXX_LINK_FLAGS  "-std=c++11")
```

Comment:
Instead of using `set(CMAKE_CXX_LINK_FLAGS  "-std=c++11")` see if 

```
set(CMAKE_CXX_STANDARD 11)
set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

works

Reply:
Thanks, @arthursoprana. It works! Please note that I had to change it also inside `hookman_generator.py`.